pkg/gofr/gofr.go

```diff
@@ -374,11 +351,50 @@ func (a *App) getExporter(name, host, port, url string) (sdktrace.SpanExporter,
 
 		exporter = NewExporter(url, logging.NewLogger(logging.INFO))
 	default:
-		a.container.Errorf("unsupported trace exporter: %s", name)
+		if name == "" {
+			a.container.Errorf("TRACE_EXPORTER env is missing")
+		} else {
+			a.container.Errorf("unsupported trace exporter: %s", name)
+		}
 	}
 
 	return exporter, err
 }
+func (a *App) buildOtlp(url, host, port, exporter string) (sdktrace.SpanExporter, error) {
```

Comment:
OTLP is an acronym, and most people are not aware of what this acronym could mean

Add comments even if not exported, and respect acronym or Go initialism

```suggestion
// buildOTLP builds OpenTelemetry Protocol 
func (a *App) buildOTLP(url, host, port, exporter string) (sdktrace.SpanExporter, error) {
```

Or do that


```suggestion
func (a *App) buildOpenTelemetryProtocol(url, host, port, exporter string) (sdktrace.SpanExporter, error) {
```